charts/sn-console/templates/console/streamnative-console-ingress.yaml

```diff
@@ -0,0 +1,62 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+{{- if .Values.ingress.enabled }}
+
+{{/* COMMENT */}}
+
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: "{{ template "sn_console.fullname" . }}-{{ .Values.component }}-ingress"
+  namespace: {{ template "sn_console.namespace" . }}
+  labels:
+    {{- include "sn_console.standardLabels" . | nindent 4 }}
+    component: {{ .Values.component }}
+  annotations:
+{{- if .Values.ingress.tls.enabled }}
+    # don't do ssl-redirect if ssl is already terminated at the load balancer level
+    ingress.kubernetes.io/ssl-redirect: "true"
+{{- if and .Values.ingress.public_issuer.enabled }}
+    kubernetes.io/tls-acme: "true"
+    cert-manager.io/issuer: "{{ .Values.ingress.public_issuer.issuer }}"
+{{- end }}
+{{- else }}
+    ingress.kubernetes.io/ssl-redirect: "false"
+{{- end }}
+    kubernetes.io/ingress.class: nginx
+  {{- with .Values.ingress.annotations }}
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+spec:
+{{- if .Values.ingress.tls.enabled }}
+  tls:
+    - hosts:
+      - {{ .Values.ingress.host }}
+      secretName: {{ .Values.component }}-ingress
+{{- end }}
+  rules:
+    - host: {{ .Values.ingress.host }}
+      http:
+        paths:
```

Comment:
With apiVersion `networking.k8s.io/v1`, there are some fields that need to update.
- Add pathType, you can use `ImplementationSpecific` as default
- serviceName 
- servicePort
You can find the demo in this doc.
https://kubernetes.io/docs/concepts/services-networking/ingress/#the-ingress-resource

Reply:
Thanks

Reply:
Fixed